addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -150,13 +150,14 @@ registerModel({
                 isMemberListOpened: true,
             });
         },
-        /**
-         * @param {Event} ev
-         */
-        onFocusInNewMessageFormInput(ev) {
-            if (this.exists()) {
-                this.update({ isFocused: true });
+        onComponentUpdate() {
+            if (this.isDoFocus) {
+                this.update({ isDoFocus: false });
+                if (this.newMessageFormInputView) {
+                    this.newMessageFormInputView.update({ doFocus: true });
+                }
```

Comment:
Strange that the "composer" case not handled here, but I guess it directly uses the field on composer. But then why the other case doesn't directly use the field on autocomplete input?

Reply:
It's not normal to have a "do focus" chain at component level here, when it should be done in models as long in possible instead, and only the very last component, the one that has the input, should handle it.

Reply:
Done (immediate focus input view on calling `focus()`)